src/index.ts

```diff
@@ -746,6 +746,25 @@ const handleFilterOperator = <
 			filterStr = `${operator.slice(1)}(${alias}:${filterStr})`;
 			return addParentKey(filterStr, parentKey, '/');
 		}
+		case '$fn': {
+			const filterx = filter as FilterType<typeof operator, T>;
+			const scope = filterx.$scope;
+			const method = filterx.$method;
+			const args = (filterx.$args ?? []).map((v) => `${escapeValue(v)}`);
```

Comment:
You can probably avoid the `map` call in the no args case by changing to:
```suggestion
			const args = filterx.$args?.map((v) => `${escapeValue(v)}`) ?? [];
```